cli/index.js

```diff
@@ -11,9 +11,7 @@ function main () {
   const argsOptions = parseArgs(process.argv.slice(2), {
     boolean: ['h', 'help', 'v', 'version', 'i', 'docker'],
     string: ['config'],
-    default: {
-      config: 'backstop.json'
```

Comment:
@klodoma @maxfenton @dgrebb 

FWIW:  if we change this to `backstop` then require() will resolve this as a `.js` or a `json` file on its own.   This would be the absolute simplest change.  I don't think any other changes would be needed.

Reply:
@garris 

Please have a look at this PR. I'll add more comments in the PR. 
https://github.com/garris/BackstopJS/pull/1566

So probably we'll abandon this current PR.

Reply:
I will test both today, but I think the docs on the new PR still suggest a `backstop.config.js` filename and I'm not clear if resolve will find a file with `.config` in it.

I do like that this PR makes it clear what filenames are allowed and what order of priority they would take.

Reply:
I find this PR to be more elegant than v2 which adds a `useJs` flag that seems extraneous. I think the logic in this PR handles the case where someone has added both a `backstop.js` and `backstop.json` in the directory (the json is used because of array priority) and this PR handles more filenames and shows the order of them more explicitly.

That's just my opinion, but I'd vote for this PR over the other one, even though `resolve` can spot `.json` and `.js`

Reply:
(And I really appreciate the work in both PR's @klodoma!)